doc/samtools-mpileup.1

```diff
@@ -482,7 +483,43 @@ The
 .B -E
 option can be used to make it ignore the contents of the BQ:Z tag and
 force it to recalculate the BAQ scores by making a new alignment.
-
+.PP
+.SH EXAMPLES
+.PP
+Using range:
+.PP
+with implicit index files in1.bam.<ext> and in2.sam.gz.<ext>,
+.IP
+.B samtools mpileup in1.bam in2.sam.gz -r <name>:<range>
+.PP
+with explicit index files,
+.IP
+.B samtools mpileup in1.bam in2.sam.gz <in1 index path> <in2 index path> -X -r <name>:<range>
+.PP
+with fofn having input file names, with implicit index files present along with inputs,
+.IP
+.B samtools mpileup -b fofn -r <name>:<range>
+.PP
+Using flags:
+.PP
+to get reads with flags READ2 or REVERSE and not having any of SECONDARY,QCFAIL,DUP,
+.IP
+.B samtools mpileup --rf READ2,REVERSE in.sam
+.R or
+.B samtools mpileup --rf 144 in.sam
+.PP
+to get reads with flag SECONDARY,
+.IP
+.B samtoos mpileup --rf SECONDARY --ff QCFAIL,DUP in.sam
+.R or
+.B samtools mpileup --rf 256 --ff 1536 in.sam
+.PP
```

Comment:
This applies to all other examples too, but just cherry picking one to discuss.

All the examples in other man pages, including the old examples here before we removed them, use `.EX` and `.EE` to mark text as example, rather than displaying in bold.  This also makes it fixed-width font on the web. See http://www.htslib.org/doc/samtools-view.html#EXAMPLES for examples.

We don't appear to have consistent indenting, but it looks like we're mostly doing `.EX 2` or `.EX 4`.  The old mpileup man was `.EX 2` so maybe just stick with that.  It's sufficient either way.  

I also fixed typo of "samtoos" above, removed the second "or" example for -rf 256 etc as we don't need to labour the point of using symbolic strings or numerics twice and less to read is better IMO.  Finally I prefer new paragraphs starting with a capital letter.  

So for example:

```
.PP
Using flags:
to get reads with flags READ2 or REVERSE and not having any of SECONDARY,QCFAIL,DUP,
.EX 2
samtools mpileup --rf READ2,REVERSE in.sam
.EE
or
.EX 2
samtools mpileup --rf 144 in.sam
.EE
To get reads with flag SECONDARY,
.EX 2
samtools mpileup --rf SECONDARY --ff QCFAIL,DUP in.sam
.EE
```

This renders in the terminal as:

```
       Using  flags:  to  get  reads  with  flags  READ2  or  REVERSE  and  not having any of SEC-
       ONDARY,QCFAIL,DUP,

         samtools mpileup --rf READ2,REVERSE in.sam

       or

         samtools mpileup --rf 144 in.sam

       To get reads with flag SECONDARY,

         samtools mpileup --rf SECONDARY --ff QCFAIL,DUP in.sam
```

*Some* other man pages have bullet point sections (eg http://www.htslib.org/doc/samtools-view.html#EXAMPLES mentioned already) while others do not (http://www.htslib.org/doc/samtools-fasta.html#EXAMPLES).   Given we don't really have a specific style, I'll leave that up to you, but if you feel it helps with the structure then take a look at samtools-view.1 for how this is done.  IMO it looks a bit ugly in the text terminal, but can still be helpful to break up lots of separate examples and delineate where one finishes and the next starts if we have multiple paragraphs.

Reply:
done.